docs/app/pages/components/components-sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -1,133 +1,137 @@
 <table class="table table-hover">
     <thead>
-        <tr uxColumnSorting>
-            <th></th>
-            <th class="clickable"
-                uxFocusIndicator
-                tabindex="0"
-                [attr.aria-label]="getColumnAriaLabel('Name', nameColumn)"
-                [attr.aria-sort]="nameColumn.state"
-                (click)="changeState('Name', nameColumn)"
-                (keydown.enter)="changeState('Name', nameColumn)">
+    <tr uxColumnSorting>
```

Comment:
Seems like the indentation got a bit messed up here.

Reply:
Thanks! Should now be fixed